src/ai/backend/manager/server.py

```diff
@@ -490,6 +491,113 @@ async def monitoring_ctx(root_ctx: RootContext) -> AsyncIterator[None]:
         await ectx.cleanup()
 
 
+@actxmgr
+async def hanging_sessions_scanner_ctx(root_ctx: RootContext) -> AsyncIterator[None]:
```

Comment:
It's a minor thing, but we could just name it `hanging_session_scanner_ctx` (no plural for "session" in the middle of the composite word).

Reply:
Resolved: 8be780b530fb0c5b268ea00df67a74cac8200284